Sources/Prometheus/Counter.swift

```diff
@@ -0,0 +1,90 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftPrometheus open source project
+//
+// Copyright (c) 2018-2023 SwiftPrometheus project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftPrometheus project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import Atomics
+import CoreMetrics
+
+public final class Counter: Sendable {
+    let intAtomic = ManagedAtomic(Int64(0))
+    let floatAtomic = ManagedAtomic(Double(0).bitPattern)
```

Comment:
nit but it's weird to call a Double property "float"; yeah "floating point" but it's a double -> 
```suggestion
    let doubleAtomic = ManagedAtomic(Double(0).bitPattern)
```